cirq-superstaq/cirq_superstaq/service.py

```diff
@@ -578,13 +578,16 @@ def qscout_compile(
     def cq_compile(
         self,
         circuits: Union[cirq.Circuit, Sequence[cirq.Circuit]],
+        grid_shape: Optional[Tuple[int, int]] = None,
         target: str = "cq_hilbert_qpu",
         **kwargs: Any,
     ) -> css.compiler_output.CompilerOutput:
         """Compiles and optimizes the given circuit(s) to the target CQ device.
 
         Args:
             circuits: The circuit(s) to compile.
+            grid_shape: Optional fixed dimensions for the rectangular qubit grid
+            (by default the actual qubit layout will be pulled from the hardware provider)
             target: String of target CQ device.
```

Comment:
```suggestion
            target: String of target CQ device.
            grid_shape: Optional fixed dimensions for the rectangular qubit grid (by default the
                actual qubit layout will be pulled from the hardware provider).
```
(reordering according to previous suggestion and nit formatting)